tutorials/lunar_lander.ipynb

```diff
@@ -555,17 +555,25 @@
     }
    ],
    "source": [
+    "from dask.distributed import Client, LocalCluster\n",
+    "\n",
     "start_time = time.time()\n",
     "total_itrs = 300\n",
     "workers = 2  # Adjust the number of workers based on your available CPUs.\n",
     "\n",
+    "client = Client(LocalCluster(\n",
```

Comment:
I think we can just do `client = dask.Distributed.Client(n_workers=workers)`

Reply:
We can take out `process=True`, but I think we should be explicit about threads per worker according to [these lines](https://github.com/dask/distributed/blob/716d5260ce8b9f8c5e46fbdfba54f0805f40fd9a/distributed/deploy/local.py#L209-L211), since pyribs is supposed to be single-threaded

Reply:
Good catch. We can still take out the call to LocalCluster as kwargs are passed from client to localcluster